onboarding.md

```diff
@@ -0,0 +1,27 @@
+# Seattle Flu Study onboarding
+
+## Adding new database users
+Before adding a new study member to the production database, confirm the DTUA is executed with Robin.
+
+Our convention for creating new usernames is first initial plus last name.
+
+Once you've created new database credentials with the appropriate grants, send them in an encrypted email to the new study member [with the word "secure" (unquoted) in the subject](https://centernet.fredhutch.org/content/dam/centernet/u/center-it/Ignite2/Fred%20Hutch%20Email%20Encryption%20Instructions%202018-11-20.pdf).
```

Comment:
Oh interesting. I was using Outlook on Office365 and clicking the Encrypt button.

Reply:
> Might be good to also mention the `id3c user create` command.

That would be good to mention! I didn't even know it existed....